docs/quick-start/write-data.txt

```diff
@@ -0,0 +1,97 @@
+.. _laravel-quick-start-write-data:
+
+=====================
+Write Data to MongoDB
+=====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: test connection, runnable, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create the Function to Insert Data
+
+      Update the ``store()`` method ``MovieController.php``, located in
+      the ``app/Http/Controllers`` directory to insert request data
+      into the database as shown in the following code:
+
+      .. code-block:: php
+
+         public function store(Request $request)
+         {
+             $data = $request->all();
+             $movie = new Movie();
+             $movie->fill($data);
+             $movie->save();
+         }
+
+   .. step:: Add an API Route that Calls the Controller Function
+
+      Import the controller and add an API route that calls the ``store()``
+      method in the ``routes/api.php`` file:
+
+      .. code-block:: php
+
+         use App\Http\Controllers\MovieController;
+
+         // ...
+
+         Route::resource('movies', MovieController::class)->only([
+           'store'
+         ]);
+
+
+   .. step:: Update the Model Fields
+
+      Update the ``Movie`` model in the ``app/Models`` directory to
+      specify the fields that the ``fill()`` method populates as shown in the
+      following code:
+
+      .. code-block:: php
+         :emphasize-lines: 4
+
+         class Movie extends Model
+         {
+            protected $connection = 'mongodb';
+            protected $fillable = ['title', 'year', 'runtime', 'imdb', 'plot'];
+         }
+
+   .. step:: Post a Request to the API
+
+      Create a file called ``movie.json`` and insert the following data:
+
+      .. code-block:: json
+
+         {
+           "title": "The Laravel MongoDB Quick Start",
+           "year": 2024,
+           "runtime": 15,
+           "imdb": {
+             "rating": 9.5,
+             "votes": 1
+           },
+           "plot": "This movie entry was created by running through the Laravel MongoDB Quick Start tutorial."
+         }
+
+      Send the JSON payload to the endpoint as a ``POST`` request by running
+      the following command in your shell:
+
+      .. code-block:: bash
+
+         curl -H "Content-Type: application/json" --data @movie.json http://localhost:8000/api/movies
+
+   .. step:: View the Data
+
+      Open ``http://127.0.0.1:8000/browse_movies`` in your web browser to view
```

Comment:
S: nit, but since this is a URL and it's clickable in view-data.txt, could be helpful to also make this link clickable:

```suggestion
      Open http://127.0.0.1:8000/browse_movies in your web browser to view
```